src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -813,7 +813,9 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
                     dependenciesToCommit.AddRange(coherencyUpdate.deps);
                 }
 
-                await darc.CommitUpdatesAsync(targetRepository, newBranchName, dependenciesToCommit.Select(du => du.To).ToList(), message.ToString());
+                List<GitFile> commitedFiles = await darc.CommitUpdatesAsync(targetRepository, newBranchName, dependenciesToCommit.Select(du => du.To).ToList(), message.ToString());
+
+                UpdatePRDescriptionDueConfigFiles(commitedFiles, description);
```

Comment:
I think it be cleaner to move CalculatePRDescription below the CommitUpdatesAsync and pass in the original update, the coherency update, and the committed files.

Reply:
Yeah, definitely better. I only realized that after I coded it. I'll modify the code.